@xen-orchestra/web/src/components/account-menu/AccountMenu.vue

```diff
@@ -10,7 +10,7 @@
     </template>
 
     <MenuItem v-for="link in links" :key="link.href" class="icon" :icon="link.icon">
-      <UiLink size="small" class="link typo-body-bold-small" :href="link.href">
+      <UiLink size="small" class="link typo-body-bold-small" :href="link.href" :target="link.target">
         {{ link.label }}
       </UiLink>
     </MenuItem>
```

Comment:
If using my suggestion, the code will become:

```suggestion
    <MenuItem v-for="(link, index) in links" :key="index">
      <UiLink size="small" class="link typo-body-bold-small" v-bind="link.props">
        {{ link.label }}
      </UiLink>
    </MenuItem>
```
Remember to delete unused CSS classes in the CSS section.